newsfragments/47761.significant.rst

```diff
@@ -25,6 +25,9 @@ If you have these configuration options in your ``airflow.cfg`` file, you need t
   * [ ] Code interface changes
 
 * Migration rules needed
+
+  * ``airflow config lint``
```

Comment:
Dont we have a precommit to check these things?

It should cover this part right?

Reply:
We have a pre-commit to check most of the format regulations. But it's not easy to cover all cases, and might not be worth the time